MiniApp/Classes/JavascriptBridge/bridge.js

```diff
@@ -114,6 +114,22 @@ MiniAppBridge.requestPermission = function(permissionType) {
     );
   });
 };
+
+/**
+ Below code will override the navigator.geolocation.getCurrentPosition method
+ */
+navigator.geolocation.getCurrentPosition = function(success, error, options) {
+    return MiniAppBridge.exec(
+        "getCurrentPosition",
+        null,
```

Comment:
There are some options that can be passed to `getPosition` ([see here](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition)). Does it make sense to support any of them?

Probably `maximumAge` isn't relevant, right? But not sure about `timeout` and `enableHighAccuracy`. Looks like there's a `CLLocationManager#desiredAccuracy`. Can this be changed after initialization?

Reply:
maximumAge and timeout are irrelevant because the location info will be returned immediately. I have updated the logic to use `enableHighAccuracy` now. If `enableHighAccuracy` is true then CLLocationManager#desiredAccuracy set as `kCLLocationAccuracyBest` and if enableHighAccuracy is false then CLLocationManager#desiredAccuracy set as `kCLLocationAccuracyHundredMeters`